app/src/main/java/moe/apex/rule34/largeimageview/LargeImageView.kt

```diff
@@ -88,9 +88,9 @@ private fun isUsingWiFi(context: Context): Boolean {
 @SuppressLint("UnusedMaterial3ScaffoldPaddingParameter")
 @Composable
 fun LargeImageView(
+    shouldShowLargeImage: MutableState<Boolean>,
```

Comment:
I think the name of this is a little verbose when already inside the LargeImageView compared to `visible` but not a big deal and it's at least consistent with what we call it elsewhere. No need to change unless you want to.